playwright/tests/responsevalidation.spec.ts

```diff
@@ -0,0 +1,50 @@
+import {test, expect} from '@playwright/test';
+import {partialBody} from '../data/response-body';
+const fullBody = require('../data/users')
```

Comment:
Maybe you could use import here as well.

Reply:
Done, thank you for the suggestion!

Reply:
Another solution could be, if you use a `tsconfig.json` file, you can set it so u can import json files as well.
We have this file in our repository, and the option is set. (this could be considered later on).